cfe_internal/update/update_bins.cf

```diff
@@ -68,18 +82,6 @@ bundle agent cfe_internal_update_bins
 
       "admin_file"      string => "/tmp/admin_file";
 
-    windows.enterprise::
-
-      "stop_signal"              string => "kill",
-      comment => "Define a stop signal for Windows",
-      handle => "cfe_internal_update_bins_vars_stop_signal_windows";
-
-    !windows.enterprise::
-
-      "stop_signal"              string => "term",
-      comment => "Define a stop signal for *NIX system",
-      handle => "cfe_internal_update_bins_vars_stop_signal_unix";
-
```

Comment:
This looks like it doesn't belong here, but I take it that ``stop_signal`` is not used anywhere. This should be made into a separate commit in case we need to revert either fix later.